### PR TITLE
feat: dynamic filler address repository

### PR DESCRIPTION
**Pull request by ConjunctiveNormalForm**

### CODE REVIEW

File: lib/repositories/filler-address-repository.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+  static create(documentClient: DynamoDBDocumentClient): FillerAddressRepository {
+    this.log = Logger.createLogger({
+      name: 'FillerAddressRepository',
+      serializers: Logger.stdSerializers,
+    });
+
+    const addressTable = new Table({
+      name: DYNAMO_TABLE_NAME.FILLER_ADDRESS,
+      partitionKey: 'pk', // generic partition key name to support both filler and address
```
Comment (codyborn) [inline lib/repositories/filler-address-repository.ts:27]: It looks like DynamoDB supports secondary indexes so we don't need to store two separate records just for indexing:
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html

Comment (ConjunctiveNormalForm) [reply lib/repositories/filler-address-repository.ts:27]: Is the suggestion that we store one `filler<>address` relation per item, make one the partition key and the other the primary key of the index? I feel like that's more complex and doesn't bring performance benefits 🤔

File: lib/quoters/WebhookQuoter.ts [hunk 2]
```diff
@@ -241,6 +243,10 @@ export class WebhookQuoter implements Quoter {
       );
 
       //if valid quote, log the opposing side as well
+      // do not await
```
Comment (marktoda) [inline lib/quoters/WebhookQuoter.ts:246]: why not? just to not block? does this mean we could get a rejection here and not see it?

File: lib/quoters/WebhookQuoter.ts [hunk 3]
```diff
@@ -241,6 +243,10 @@ export class WebhookQuoter implements Quoter {
       );
 
       //if valid quote, log the opposing side as well
```
Comment (marktoda) [inline lib/quoters/WebhookQuoter.ts:245]: this comment should be below

**Review by marktoda — APPROVED** [review]

File: lib/quoters/WebhookQuoter.ts (hunk 2 above)
Comment (ConjunctiveNormalForm) [reply lib/quoters/WebhookQuoter.ts:246]: yeah just to have as little latency as possible on the hot path. Even if there are occasional rejects I think that's fine?

**Review by marktoda — APPROVED** [review]

**Pull request merged**